fms/testing/_internal/model_test_suite.py

```diff
@@ -66,10 +66,11 @@ def model(self, uninitialized_model: nn.Module):
         params = sorted(sd.keys())
         for key in params:
             parameter = sd[key]
-            values = torch.randn_like(parameter)
-            values -= 0.5
-            values /= 20.0
-            parameter.copy_(values)
+            if parameter.dtype != torch.int:
+                values = torch.randn_like(parameter)
+                values -= 0.5
+                values /= 20.0
+                parameter.copy_(values)
```

Comment:
what happens here to the int parameters (`qweight` and `qzeros`)? how are they initialized?

Reply:
They are initialized to whatever the model is after `reset_paramters` is called I believe. For this, we want to ignore them i would think, unless you have a suggestion in terms of Mock test values for this.

Reply:
fixed in latest commit